region/crateria/central.json

```diff
@@ -2599,6 +2648,66 @@
                   "requires": []
                 }
               ]
+            },
+            {
+              "id": 3,
+              "strats": [
+                {
+                  "name": "G-Mode Morph to Bomb the Bomb Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Bombs"
+                  ],
+                  "note": [
+                    "Overload PLMs using the scroll block directly above the bomb block leading down to the item.",
+                    "When the bomb blocks turn to air, lay bombs as Samus falls.",
+                    "After landing on the first platform, quickly unmorph and use X-Ray to cancel G-mode before the bombs explode."
+                  ]
+                },
+                {
+                  "name": "G-Mode Morph to Power Bomb the Bomb Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "PowerBomb",
+                    {"ammo": {"type": "PowerBomb", "count": 5}}
+                  ],
+                  "note": [
+                    "Use Power Bomb horizontal boosts to move toward the left side of the room in artificial morph.",
+                    "Roll off the second-to-last blue platform and lay a Power Bomb against the wall to get into range of the bomb blocks.",
```

Comment:
Could be helpful to say that this is underneath the platform pirate.
And to be careful bumping the wall pirate when bomb jumping over, or jumping into any spawned lazers.